src/aura/HH_ContactCard/HH_ContactCard.cmp

```diff
@@ -18,7 +18,7 @@
                                    xlinkHref="/resource/SLDS/1_0_3/assets/icons/standard-sprite/svg/symbols.svg#contact" />
                         </div>
                         <div class="slds-media__body">
-                            <a href="#">{!v.contact.FirstName}&nbsp;{!v.contact.LastName}</a>
+                            <a href="{! '/' + v.contact.Id}" target="_blank">{!v.contact.FirstName}&nbsp;{!v.contact.LastName}</a>
```

Comment:
I saw your Chatter post regarding this, so I know you were having issues without the `target="\_blank"` attribute.  But, does using this attribute work in LEX?  I thought LEX gets upset when you try to pop it open in a new window.